test/e2e/framework/kubernetes/no-crashes.go

```diff
@@ -14,6 +14,7 @@ type EnsureStableCluster struct {
 	LabelSelector      string
```

Comment:
nit: The struct name is a little misleading. When you say cluster, assumption is everything in a cluster. What I think this actually is a `EnsureStableComponent`. You can use this struct to check specific components in your cluster - like `retina` or `npm`, etc.

Reply:
Valid point, I updated the name.